packages/react/src/utils/checkInternetAccess.ts

```diff
@@ -5,19 +5,21 @@ import { externalUrls } from "../constants/externalUrls";
 import { analyzeSolution } from "../store/actions/backend";
 import { setCurrentMessageUpdate } from "../store/actions/error";
 
-export const checkInternetAccess  = async (solutionPath: string, dispatch: Dispatch) => {
-  const haveInternet:boolean = await window.backend.checkInternetAccess();
+export const checkInternetAccess = async (solutionPath: string, dispatch: Dispatch) => {
+  const haveInternet: boolean = await window.backend.checkInternetAccess();
   if (!haveInternet) {
     dispatch(
-      setCurrentMessageUpdate([{
-        messageId: uuid(),
-        groupId: "accessPrereqFailed",
-        type: "error",
-        header: "Unable to access S3",
-        content: "Please check your internet connection",
-        buttonText: "View prerequisites",
-        onButtonClick: () => window.electron.openExternalUrl(externalUrls.prereq)
-      }])
+      setCurrentMessageUpdate([
+        {
+          messageId: uuid(),
+          groupId: "accessPrereqFailed",
+          type: "error",
+          header: "Unable to access S3",
+          content: "Please check your internet connection",
+          buttonText: "View prerequisites",
```

Comment:
Is this button visible to the user? If yes, should it be View Prerequisites instead?

Reply:
These changes were merged from main into this branch/pr. @cslong can provide more details.

Reply:
@marknfawaz I took a look at some of the other buttons in the application and it looks like we do not upcase each word